.pipelines/07-processing-data-os-cmd.yml

```diff
@@ -0,0 +1,57 @@
+# Continuous Integration (CI) pipeline that orchestrates the training, evaluation, and registration of the model.
+
+resources:
+  containers:
+  - container: mlops
+    image: public/mlops/tensorflow:latest
+    endpoint: mlopstensorflowacr
+
+pr: none
+trigger:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - ml_model/preprocessing/Dockerfile
+    - ml_model/preprocessing/preprocess_os_cmd.py
+    - ml_service/pipelines/build_data_processing_os_cmd_pipeline.py
+    - .pipelines/07-processing-data-os-cmd.yml
+    - .pipelines/variables-template.yml
+    - .pipelines/code-quality-template.yml
+
+
+variables:
+- template: variables-template.yml
+- group: devopsforai-aml-vg
+
+pool:
+  vmImage: ubuntu-latest
+
+stages:
+- stage: 'Data_CI'
+  displayName: 'Data code quality and publish preprocessing pipeline'
+  jobs:
+  - job: "Data_CI_Pipeline"
+    displayName: "Data CI Pipeline"
+    container: mlops
+    timeoutInMinutes: 0
+    steps:
+    - template: code-quality-template.yml
+    - task: AzureCLI@1
+      enabled: true
+      inputs:
+        azureSubscription: '$(WORKSPACE_SVC_CONNECTION)'
+        scriptLocation: inlineScript
+        workingDirectory: $(Build.SourcesDirectory)
+        inlineScript: |
+          set -e # fail on error
+          export SUBSCRIPTION_ID=$(az account show --query id -o tsv)
+          # Invoke the Python building and publishing a data preprocessing pipeline
+          python -m ml_service.pipelines.build_data_processing_os_cmd_pipeline
+      displayName: 'Publish Data Preprocessing OS cmd Pipeline'
+
+# Trigger_Preprocessing_Pipeline
+- template: trigger-preprocessing-pipeline.yml
```

Comment:
Added the top level AzDO build/deploy pipeline for running non python tools AML pipeline.